sanitize_html/lib/sanitize_html.dart

```diff
@@ -46,14 +46,40 @@ import 'src/sane_html_validator.dart' show SaneHtmlValidator;
 /// }
 /// ```
 ///
+/// It is furthermore possible to use the [addLinkRel] to attach a `rel="..."`
+/// property to links (`<a href="..."`). When hosting user-generated content it
+/// can be useful to [qualify outbound links][2] with `rel="ugc"`, as this lets
+/// search engines know that the content is user-generated.
+///
+/// **Example**
+/// ```dart
+/// import 'package:sanitize_html/sanitize_html.dart' show sanitizeHtml;
+///
+/// void main() {
+///   print(sanitizeHtml(
+///     '<a href="https://spam.com/free-stuff">free stuff</a>',
+///     addLinkRel: (href) => ['ugc', 'nofollow'],
```

Comment:
Can `addLinkRel` be made optional? This has broken google maps flutter web [here](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart#L384) when users [do this](https://github.com/flutter/flutter/issues/67854#issuecomment-707781063).

Thanks!